pytest_catchlog/fixture.py

```diff
@@ -16,6 +16,9 @@ class LogCaptureFixture(object):
     def __init__(self, item):
         """Creates a new funcarg."""
         self._item = item
+        # set logging levels to facilitate use without needing to import logging
+        for level in ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']:
+            setattr(self, level, getattr(logging, level))
```

Comment:
@TBeijen Sure, I see. But still, what do you think about the proposed solution?
```py
    def __getattr__(self, name):
        if name == name.upper():
            return getattr(logging, name)
        return getattr(super(...), name)
```
It seems to solve the issue in question in a generic way, without the need to know any of existing levels.